src/components/SelectComponents/SuggestionsList/SuggestionsList.tsx

```diff
@@ -54,12 +54,11 @@ export function SuggestionsList<S>(props: Props<S>) {
         passDisabledToListItems,
     } = props;
 
-    // eslint-disable-next-line react/display-name
     const renderItem = ({ key, index, style = {} }) => {
         const currentItem = suggestions[index];
         let disabled = false;
         if (typeof currentItem === 'object' && 'disabled' in currentItem) {
-            // eslint-disable-next-line dot-notation
+            // eslint-disable-next-line @typescript-eslint/dot-notation
```

Comment:
This is a temporary workaround: tests would issue an error without it.

Reply:
LGTM